nuxeo-core/nuxeo-core-api/src/main/java/org/nuxeo/ecm/core/api/trash/TrashService.java

```diff
@@ -84,6 +84,17 @@ default boolean canPurgeOrUntrash(DocumentModel doc, NuxeoPrincipal principal) {
         return canPurgeOrUntrash(Collections.singletonList(doc), principal);
     }
 
+    /**
+     * Check if trashed child documents from a given parent document are "purgeable/untrashable".
+     * Documents need to be in the trash for this to be true, in addition to the standard permission checks.
+     * @param parent the parent document.
+     * @param principal the user.
```

Comment:
don't add dot at the end of `@param`
for the return tag you can use:
` @return {@code true} if .......`
`<code>true</code>` -> `{@code}`